windows/CommonSettings.props.example

```diff
@@ -11,6 +11,10 @@
          set to the root of your Python installation. If your Python installation
          does not contain debug libraries, debug build will not work. -->
         <PythonSupport>false</PythonSupport>
+        <!-- NOTE: If Matlab support is enabled, MatlabDir (below) needs to be
+         set to the root of your Matlab installation, and Python support has to
+         be disabled. Only works in Release mode. -->
```

Comment:
If PythonSupport is enabled then the libboost_python-vc120-mt-1_59.lib dependency requirement is transferred to the matcaffe compilation. Then, I get this linking error (LINK : fatal error LNK1104: cannot open file 'libboost_python-vc120-mt-1_59.lib'), which probably can be solved with some conditions in the configuration file, but anyway, the functionalities from that dependence won't be used in practice in Matlab, so the easiest solution is just to disable PythonSupport when MatlabSupport is enabled.

Regarding the release/debug issue, I think there's a misunderstanding here. It works only in release, but not in debug. It is possible to compile this matcaffe project in debug with the Matlab libraries (libmx.lib;libmex.lib;libmat.lib) as dependencies, but then when testing the generated mex file in Matlab, it says "invalid MEX-file", which doesn't happen when compiled in release mode. Not sure why, but I guess this happens because those Matlab libraries are only available in release mode and there might be some kind of incompatibility with the rest of dependencies because of that.